.travis.yml

```diff
@@ -1,14 +0,0 @@
-language: php
-
-php: 
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - hhvm
-
-before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev
```

Comment:
@pleckey 

No need for this, there are no dependencies. Also, composer is pre-installed, so this would be better

```yml
before_install:
  - composer self-update

install:
  - composer install --prefer-dist
```